src/crate/client/sqlalchemy/compiler.py

```diff
@@ -309,3 +309,10 @@ def limit_clause(self, select, **kw):
         Generate OFFSET / LIMIT clause, PostgreSQL-compatible.
         """
         return PGCompiler.limit_clause(self, select, **kw)
+
+    def for_update_clause(self, select, **kw):
+        # CrateDB does not support currently 'FOR UPDATE' statements.
```

Comment:
`currently` may imply that we will support it in future while its not sure if this will ever be supported
```suggestion
        # CrateDB does not support 'FOR UPDATE' statements.
```

Reply:
Thanks. Fixed.